timeless/views.py

```diff
@@ -178,6 +180,19 @@ def get_context_object_list_name(self):
         """
         return self.context_object_list_name
 
+    def sort_query(self, query):
+        ordering = request.args.get("ordering")
+        if ordering:
+            for field_name in ordering.split(","):
+                order_direction = desc if field_name.startswith("-") else asc
+                model_field = getattr(self.model, field_name.strip("-"), None)
+                if not model_field:
+                    continue
+
+                query = query.order_by(order_direction(model_field))
+
+        return query
```

Comment:
@vladarefiev I also believe sorting should take place on the Front-End, not the backend. But let's see what @emilianodellacasa says :)

Reply:
@amihaiemil didn't catch your idea about Front-End. Pls clarify?
So far as I can see, Front-end collect and send query params to back-end, back-end parse, validate and process them

Reply:
@vladarefiev yes, that's valid in the case of filtering, for instance, but ordering can very well be done in the front-end (simple JavaScript ordering divs, no need to call the backend for that, usually)

Reply:
@amihaiemil 🤔what's about pagination, if front-end has only `N` rows, how could we sort it ? :)

Reply:
@vladarefiev you can mantain the JS filter and apply it before displaying the second page, maybe? 

But anyway, it's not a critical issue, it can also be on the backend. Trade offs :)

Reply:
@amihaiemil I've never faced with this solution before. Let's leave it in this way and improve it further if will be needed

Reply:
@vladarefiev @amihaiemil I prefer this approach, I have never seen ordering on front end... actually now that I remember I did see front end ordering on a large enterprise application and it SUCKED! :-)